hexbot.py

```diff
@@ -87,12 +87,24 @@ def send_whisper(sender, message):
 
 def parse_time(message):
     time_segs = message.split(':')
-    if not (len(time_segs) == 3 or len(time_segs) == 2):
+    if not (len(time_segs) >= 4 or len(time_segs) <= 2):
```

Comment:
I think this should `<= 4` and `>= 2`

Can you try testing this on a REPL with all our possible time formats?